docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[release-notes-header-8.14.0]]
+== 8.14
+
+[discrete]
+[[release-notes-8.14.0]]
+=== 8.14.0
+
+[discrete]
+[[features-8.14.0]]
+==== New features
+* Introduces Attack discovery, a new feature that uses AI to identify potential attacks and help you quickly triage multiple alerts ({kibana-pull}181818[#181818]).
+* Creates the **Asset criticality** page within the **Manage** menu, which lets you assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
```

Comment:
```suggestion
* Creates the **Asset criticality** page within the **Manage** menu, which lets you bulk assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
```
Slightly more precise, since users can already assign asset criticality without using this new page.